### PR TITLE
fix bug on empty points in point in polygon op

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[PIP-Empty-Fix](https://trello.com/c/gPE6XBAK/417-pip-empty-fix)

**Comment by ChristianBeilschmidt** [comment]
[Point-in-Polygon: Leere Punktmenge](https://trello.com/c/liJtQvP0/416-point-in-polygon-leere-punktmenge)

File: operators/src/processing/point_in_polygon.rs [hunk 1]
```diff
@@ -133,6 +133,10 @@ impl PointInPolygonFilterProcessor {
         polygons: &MultiPolygonCollection,
         thread_pool: &ThreadPoolContext,
     ) -> Vec<bool> {
+        if points.is_empty() {
```
Comment (michaelmattig) [inline operators/src/processing/point_in_polygon.rs:136]: wouldn't it make more sense to check this before spawning this task in the first place?

Comment (ChristianBeilschmidt) [reply operators/src/processing/point_in_polygon.rs]: Yes

11d2c98

**Review by michaelmattig — APPROVED** [review]

**Pull request merged**